examples/peer-and-content-routing/README.md

```diff
@@ -10,31 +10,28 @@ Content Routing is the category of modules that offer a way to find where conten
 
 This example builds on top of the [Protocol and Stream Muxing](../protocol-and-stream-muxing). We need to install `libp2p-kad-dht`, go ahead and `npm install libp2p-kad-dht`. If you want to see the final version, open [1.js](./1.js).
 
-First, let's update our bundle to support Peer Routing and Content Routing.
+First, let's update our config to support Peer Routing and Content Routing.
 
 ```JavaScript
-class MyBundle extends libp2p {
-  constructor (_options) {
-    const defaults = {
-      modules: {
-        transport: [ TCP ],
-        streamMuxer: [ Mplex ],
-        connEncryption: [ SECIO ],
-        // we add the DHT module that will enable Peer and Content Routing
-        dht: KadDHT
-      },
-      config: {
-        dht: {
-          // dht must be enabled
-          enabled: true,
-          kBucketSize: 20
-        }
-      }
+const Libp2p = require('libp2p')
+const KadDHT = require('libp2p-kad-dht')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [ TCP ],
+    streamMuxer: [ Mplex ],
+    connEncryption: [ SECIO ],
+    // we add the DHT module that will enable Peer and Content Routing
+    dht: KadDHT
+  },
+  config: {
+    dht: {
+      // dht must be enabled
+      enabled: true,
+      kBucketSize: 20
```

Comment:
We don't use `kBucketSize` in the code, we should probably just leave it out here. Most users won't need to change the default.